tests/test.validator.js

```diff
@@ -85,6 +85,18 @@ describe('Validator', function() {
     assert.equal(parsedJSON.summary.warnings, 0);
   });
 
+  it('should send JSScanner messages to the collector', (done) => {
```

Comment:
I wonder about this, if it might be brittle when we get to the point of having more tests. E.g. will the count increase for example. I wonder if we should have a way to specify a specific test or list of tests or whether we should instead ignore this and work on the basis that the more granular level of tests will suffice. I'm leaning towards the latter approach.